### PR TITLE
Use mainline Linux load tracking events names

### PR DESCRIPTION
**Pull request by douglas-raillard-arm**

### CODE REVIEW

**Comment by douglas-raillard-arm** [comment]
Actually, the kernel does not have a `sched_pelt_se` event, it only has a `sched_pelt_se` tracepoint. That said, it's probably a better idea to give the same name to the trace point and associated event.

TODO: update tools/kmodules/ to emit the `sched_pelt_se` name rather than `sched_load_se`

File: lisa/analysis/load_tracking.py [hunk 1]
```diff
@@ -61,7 +71,7 @@ def _columns_to_drop(cls, event):
         """
         The extra columns not shared between trace event versions
         """
-        if event in ['sched_load_cfs_rq', 'sched_load_se']:
+        if event in ['sched_load_cfs_rq', 'sched_load_se', 'sched_pelt_se']:
```
Comment (qais-yousef) [inline lisa/analysis/load_tracking.py]: Can't you use the new _SCHED_PELT_SE_NAMES instead here?

File: lisa/analysis/load_tracking.py [hunk 2]
```diff
@@ -71,7 +81,7 @@ def _columns_to_drop(cls, event):
         """
         The extra columns not shared between trace event versions
         """
-        if event in ['sched_load_cfs_rq', 'sched_load_se', 'sched_pelt_se']:
+        if event in [*cls._SCHED_PELT_CFS_NAMES, 'sched_load_se', 'sched_pelt_se']:
```
Comment (qais-yousef) [inline lisa/analysis/load_tracking.py]: Same here. Why not juse _SCHED_PELT_CFS_NAMES?

File: lisa/analysis/load_tracking.py (hunk 1 above)
Comment (douglas-raillard-arm) [reply lisa/analysis/load_tracking.py]: No, `sched_load_cfs_rq` is part of `_SCHED_PELT_CFS_NAMES`, so it seems to really depend on the specific event. If you have examples of them, we can sort out what fields they actually have.

File: lisa/analysis/load_tracking.py (hunk 2 above)
Comment (douglas-raillard-arm) [reply lisa/analysis/load_tracking.py]: `_SCHED_PELT_CFS_NAMES` does not contain `sched_load_se`/`sched_pelt_se`

**Pull request merged**